### PR TITLE
Transfered notebook stripping functions

### PR DESCRIPTION
**Pull request by antoine-daigneault-demers**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from setuptools import find_packages, setup
+
+SRC_DIR = "src"
+
+setup(
+    name="notebook_utils",
+    description="Functions and utilities for machine learning and data science.",
+    author="Computer Research Institute of Montreal (CRIM)",
+    version="0.0.1",
```
Comment (fmigneault) [inline setup.py]: add the license entry

**Review by fmigneault — COMMENTED** [review]
Code looks fine. 
You should add a `requirements.txt` file. 
Please also add a `CHANGES` (similar to https://github.com/Ouranosinc/Magpie/blob/master/CHANGES.rst) to track any update.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
```
Comment (fmigneault) [inline requirements.txt:4]: Please provide more generic requirements.
Only the minimal (& top-level) packages required by this tool should be added. 
Strict definitions will make it hard to install it within other environments that would not match specifically those versions.

**Review by fmigneault — APPROVED** [review]

**Review by fmigneault — CHANGES_REQUESTED** [review]
adjust requirements

File: CHANGES.rst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+0.1.1
```
Comment (fmigneault) [inline CHANGES.rst]: Any reason why this value is used as the first version?
Seems odd to have a patch number when there is not even an initial `0.1.0`.

**Review by fmigneault — APPROVED** [review]

File: CHANGES.rst (hunk 3 above)
Comment (antoine-daigneault-demers) [reply CHANGES.rst]: No, good catch I'll change it to 0.1.0

**Pull request merged**